starter/package.json

```diff
@@ -124,8 +124,8 @@
     "@angular/platform-browser-dynamic": "~7.2.2",
     "@angular/platform-server": "~7.2.2",
     "@angular/router": "~7.2.2",
-    "@nationalbankbelgium/stark-core": "latest",
-    "@nationalbankbelgium/stark-ui": "latest",
+    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-beta.4-f331991e.tgz",
+    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 changes back to "latest"? We should always keep it that way to ensure that whenever someone gets the Starter it will install the latest Stark ;)